docs/astro/src/content/docs/reference/global-namespaces/platform.mdx

```diff
@@ -0,0 +1,36 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Platform
+description: Platform Namespace
+---
+
+import SlintProperty  from '/src/components/SlintProperty.astro';
+import Link from '/src/components/Link.astro';
+
+The **Platform** namespace contains properties that help deal with platform specific differences.
+
+## Properties
+
+### os
+<SlintProperty propName="os" typeName="string">
+The name of the operating system. Possible values are:
+
+| Operating System | Value     |
+|------------------|-----------|
+| Windows          | `windows` |
+| macOS            | `macos`   |
+| Linux            | `linux`   |
+| Android          | `android` |
+| iOS              | `iOS`     |
+
+If the operating system is none of the above, the value of this property is the empty string.
```

Comment:
Add a note that more values may be added in the future.

I wonder if we should make it an enum.

Also i imagine our user would also want to distinguish between the web and native. We should document that when running on wasm, this does a runtime check.

Reply:
> Add a note that more values may be added in the future.

Ok

> Also i imagine our user would also want to distinguish between the web and native. We should document that when running on wasm, this does a runtime check.

I can add that in the docs.

For WASM, we could also add a `Platform.is-web-assembly` or so (not this particular name maybe, but a boolean - I'm open to suggestions here)